src/app/[locale]/(business)/(with-header)/users/[username]/_components/user-header/add-friend/actions.ts

```
@@ -3,18 +3,25 @@
 import { sendFriendRequest } from "@/domain/users";
 import { getCurrentUser } from "@/lib/auth";
```

ESLint

[error] There should be no empty line within import group

(import/order)